### PR TITLE
Add user-installation + DNSSEC disabling to DNS over Discord

### PR DESCRIPTION
**Pull request by MattIPv4**
### Summary

Update docs to match the upstream DNS over Discord docs, with user-installation (https://github.com/MattIPv4/DNS-over-Discord/pull/89) + DNSSEC disabling (https://github.com/MattIPv4/DNS-over-Discord/pull/65)

### Screenshots (optional)

N/A

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: DNS over Discord
 
 # DNS over Discord
 
-1.1.1.1 works from a Discord server, thanks to the 1.1.1.1 bot. [Invite the bot to your Discord server](https://cfl.re/3nM6VfQ) to start using DNS over Discord.
+1.1.1.1 works from a Discord server, thanks to the 1.1.1.1 bot. [Invite the bot to your Discord server](https://cfl.re/3nM6VfQ) to start using DNS over Discord. Or, [add the bot to your Discord account](https://dns-over-discord.v4.wtf/invite/user) to use it anywhere in Discord.
```
Comment (MattIPv4) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]: I suspect `https://discord.com/oauth2/authorize?client_id=613324904795996163&scope=applications.commands&integration_type=1` will want to be shortened to a new `cfl.re` URL, in replace of `https://dns-over-discord.v4.wtf/invite/user`?

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 2]
```diff
@@ -224,8 +244,9 @@ Example:
 
 ### `invite` command
 
-The `/invite` command provides the user with a quick link to invite the 1.1.1.1 DNS over Discord bot to another Discord server.
+The `/invite` command provides the user with a quick link to invite the 1.1.1.1 DNS over Discord bot to another Discord server, or to add it to a Discord account.
 The bot can be invited at any time with [https://cfl.re/3nM6VfQ](https://cfl.re/3nM6VfQ).
+The bot can also be added to accounts with [https://dns-over-discord.v4.wtf/invite/user](https://dns-over-discord.v4.wtf/invite/user).
```
Comment (MattIPv4) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]: Same deal here, I imagine this will want a new `cfl.re` URL?

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR. All PRs prior to August 12 need to be reworked due to the platform migration to Astro.

**Comment by MattIPv4** [comment]
👋 @KimJ15 I had already rebased the PR based on that :) Please reopen this

**Review by KimJ15 — APPROVED** [review]
LGTM

**Pull request merged**